freqtrade/misc.py

```diff
@@ -185,6 +190,14 @@ def build_subcommands(parser: argparse.ArgumentParser) -> None:
         action='store_true',
         dest='refresh_pairs',
     )
+    backtesting_cmd.add_argument(
+        '--export',
+        help='Export backtest results, argument are: trades\
+              Example --export trades',
+        type=str,
+        default=None,
```

Comment:
should there be a default argument?

Reply:
Do you mean the bot should export trades per default?